### PR TITLE
MAINT-42044 : fix duplicated notification number in the chat app window

### PR DESCRIPTION
**Pull request by hakermi**
This PR should fix the received notification duplicated number displayed in front of each room the in the chat app full window.

### CODE REVIEW

File: application/src/main/webapp/vue-app/components/ExoChatContactList.vue [hunk 1]
```diff
@@ -403,28 +403,30 @@ export default {
...
         if (!foundContact.lastMessage) {
           foundContact.lastMessage = {};
         }
         foundContact.lastMessage = message.data;
         foundContact.timestamp = message.ts;
-        if (this.selected){
+        if (this.selected) {
           if (this.selected.room !== foundContact.room || this.mq === 'mobile') {
-            foundContact.unreadTotal ++;
+            chatServices.getNotReadMessagesByRoom(eXo.chat.userSettings, chatConstants.CHAT_NOTIFICATION_TYPE, chatConstants.CHAT_NOTIFICATION_CATEGORY, foundContact.room, true).then((data) => {
```
Comment (ahamdi) [inline application/src/main/webapp/vue-app/components/ExoChatContactList.vue]: There is no need to send a request to the server to increment the number of total messages. We know the number of unread messages and we need just to incrementit.

File: application/src/test/specs/ExoChatContactList.test.js [hunk 2]
```diff
@@ -330,29 +330,9 @@ describe('ExoChatContactList.test.js', () => {
     expect(contact.unreadTotal).toBe(0);
   });
 
-  it('Increment unreadTotal for a room that receives a message', () => {
```
Comment (ahamdi) [inline application/src/test/specs/ExoChatContactList.test.js]: We should never remove test cases, we have to improve them.

**Review by ahamdi — CHANGES_REQUESTED** [review]

File: application/src/main/webapp/vue-app/components/ExoChatContactList.vue (hunk 1 above)
Comment (hakermi) [reply application/src/main/webapp/vue-app/components/ExoChatContactList.vue]: The event of receiving message is invoked more than once each time the user receive a message, so the number of messages will be incremented two times, thus i choose to get the exact value from the server

Comment (ahamdi) [reply application/src/main/webapp/vue-app/components/ExoChatContactList.vue]: We need to make sure we have only one event fired, that is what we need to fix

**Review by ahamdi — APPROVED** [review]

**Pull request merged**